main/options/options.cc

```diff
@@ -627,6 +627,8 @@ buildOptions(const vector<pipeline::semantic_extension::SemanticExtensionProvide
                                  "Packages which are allowed to ignore the restrictions set by `visible_to` "
                                  "and `export` directives",
                                  cxxopts::value<vector<string>>(), "<name>");
+    options.add_options(section)("packager-layer", "Valid layer names for packages.",
```

Comment:
We might want to mention something in our help output somewhere about how "repeatable" / `vector`-typed options work (i.e., that they can be provided as `--arg 1 --arg 2 --arg 3` or `--arg=1,2,3`)

That's not unique to this option—there are multiple such options like this, and we can do that in a future change. Probably for now, it's enough that the default shows `library,application` to help people understand that comma-separated layer names will be parsed.

Reply:
Added a TODO item to do so.